SIP/sip-48.md

```diff
@@ -0,0 +1,68 @@
+---
+sip:  48
+title: Alias renewal process
+description: Introducing a new ownership handling for aliases on a yearly basis
+author:   frank_the_tank
+status: Review
+type: standard
+category: Core
+created: 2022-12-10
+---
+## Abstract
+This SIP will introduce a temporary ownership of aliases which can be prolonged on a yearly basis.
+
+## Motivation
+Signum offers a simple setup to create aliases for an account. Once set/created those aliases are owned by the creator account forever. With the introducing of [SIP-44](sip-44.md)  the alias will be more powerful to use for like domain, account  or application pointer which makes a reserved alias name more valuable. In addition with [SIP-47](sip-44.md)  a new service SNS (Signum Naming Service) will be introduced which is a distributed, open, and extensible naming system based on the Signum blockchain. Given these cases, those aliases should be actively maintained by the creator/owner to flag those aliases are in use by them, otherwise another user should be able to claim the alias.
+
+## Specification
+The following changes should be done with an upcoming hard-fork to introduce the renewal process for aliases.
+
+### Database changes
+On the database level, the alias table will get a new field `renewal_time` which is an INT field and contains the timestamp until the alias is reserved for the current owner. With introduction of the needed hard-fork all aliases will get a reserved datetime of 3 months after the planned hard fork. In this case all active users have enough time to check their current alias ownerships and can create a renewal subscription if desired.
+
+### Node changes
+**Transaction Set Alias**
+
+If a user sends a setAlias transaction the following datetime handling for the alias will be executed:
+ - If the creation is before the block of the hard fork no datetime is set on `renewal_time`.
```

Comment:
These new transactions cannot be sent before the fork, I believe all this logic for before the fork block should be removed. These transactions simply are invalid before that.